book/index.nim

```diff
@@ -1,10 +1,13 @@
-include nimibook / prelude
-nbText: """# Introduction
+import nimib, nimibook
+import strutils
+nbInit
+nbUseNimibook # overrides nimib defaults with nimibooks and loads book configuration
 
-I should say something very wise about this book.
+proc readFileUntil(filename: string, text: string): string =
+  for line in filename.lines:
+    if line.startsWith(text):
+      return result
+    result &= line & '\n'
 
-Instead I will just post some nim code:
-"""
-nbCode:
-  echo "hello mdbook!"
+nbText: "../README.md".readFileUntil("<!--SKIP")
```

Comment:
Is it a good idea to have ``book`` be dependent on content outside its own folder ?

Reply:
see remarks above, but also reading README I think is rather harmless. I was planning to have distinct contents only for README.md and index.nim but in the end I did not use that feature.

Reply:
Yeah for a static Readme file it's fair game.